gdnative-core/Cargo.toml

```diff
@@ -22,8 +22,8 @@ euclid = "0.20.13"
 
 gdnative-impl-proc-macros = { path = "../impl/proc_macros", version = "=0.9.0-preview.0" }
 
-# Used for float comparisons in godot tests
-approx = { version = "0.3.2", optional = true }
+# Used for float comparisons in godot
```

Comment:
This comment can be removed since it's just a normal dependency now. Maybe also move it above `gdnative-impl-proc-macros` to line 20 along with the other normal dependencies?

Reply:
done :)